OptimizelySDK.Tests/OptimizelyJsonTest.cs

```diff
@@ -0,0 +1,129 @@
+﻿/**
+ *
+ *    Copyright 2020, Optimizely and contributors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+using Moq;
+using NUnit.Framework;
+using OptimizelySDK.ErrorHandler;
+using OptimizelySDK.Logger;
+using System;
+using System.Collections.Generic;
+
+namespace OptimizelySDK.Tests
+{
+    [TestFixture]
+    public class OptimizelyJsonTest
```

Comment:
Add a new class 
`class Field4 {
  public string inner_field1 {get; set;}
  public InnerField2 inner_field2 {get; set;}
}
class InnerField2 : List<object> {
}`
Now use json path to access field4 with this class. 

Secondly, add testcase related to `` empty json string.